### ISSUE TITLE
profile command broken with -v option

### ISSUE THREAD

**Issue by sstok** [rollerworks/SkeletonDancer#7]
```
PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Rollerworks\Tools\SkeletonDancer\QuestionsSet::communicate() must be an instance of Rollerworks\Tools\SkeletonDancer\Question, null given, called in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Configurator/TestingConfigurator.php on line 36 and defined in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/QuestionsSet.php:43
```

```
$ dancer profile -v

List profiles
=============

 // Using config file:
 // Project directory: /Users/sebastiaanstok/Sites/rollerworks/skeletondancer
 // Dancer config directory:

symfony:bundle
--------------

PHP Fatal error:  Uncaught TypeError: Argument 2 passed to Rollerworks\Tools\SkeletonDancer\QuestionsSet::communicate() must be an instance of Rollerworks\Tools\SkeletonDancer\Question, null given, called in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Configurator/TestingConfigurator.php on line 36 and defined in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/QuestionsSet.php:43
Stack trace:
#0 /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Configurator/TestingConfigurator.php(36): Rollerworks\Tools\SkeletonDancer\QuestionsSet->communicate('phpspec_shortna...', NULL)
#1 /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Cli/Handler/ProfileCommandHandler.php(190): Rollerworks\Tools\SkeletonDancer\Configurator\TestingConfigurator->interact(Object(Rollerworks\Tools\SkeletonDancer\QuestionsSet))
#2 /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Cli/Handler/ProfileCommandHandler.php(113): Rollerworks\Tools\SkeletonDancer\Cli\Handler\ProfileCommandHandler->getProfileQuestion in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/QuestionsSet.php on line 43

Fatal error: Uncaught TypeError: Argument 2 passed to Rollerworks\Tools\SkeletonDancer\QuestionsSet::communicate() must be an instance of Rollerworks\Tools\SkeletonDancer\Question, null given, called in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Configurator/TestingConfigurator.php on line 36 and defined in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/QuestionsSet.php:43
Stack trace:
#0 /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Configurator/TestingConfigurator.php(36): Rollerworks\Tools\SkeletonDancer\QuestionsSet->communicate('phpspec_shortna...', NULL)
#1 /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Cli/Handler/ProfileCommandHandler.php(190): Rollerworks\Tools\SkeletonDancer\Configurator\TestingConfigurator->interact(Object(Rollerworks\Tools\SkeletonDancer\QuestionsSet))
#2 /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/Cli/Handler/ProfileCommandHandler.php(113): Rollerworks\Tools\SkeletonDancer\Cli\Handler\ProfileCommandHandler->getProfileQuestion in /Users/sebastiaanstok/sites/rollerworks/skeletondancer/src/QuestionsSet.php on line 43
```

**Comment by sstok** [comment]
Hmm, it seems details doesn't work with code-blocks...

**Comment by sstok** [comment]
fixed by 782d3d23e759001f3f73db10aba015a6181bb2c6
